pyproject.toml

```
@@ -5,20 +5,23 @@ version = "0.10.5"
 enable_black = true
 enable_flake8 = true
 enable_isort = true
-enable_mypy = false    # TODO: eliminate errors and enable at CI
-mypy_preset = "entry"  # TODO: "strict"
+enable_mypy = true
+mypy_preset = "entry" # TODO: "strict"
+mypy_plugins = [
+  { function = "numpy.typing.mypy_plugin" },
+  { function = "pydantic.mypy" },
+]
 line_length = 88
 py_version = "py311"
 isort_known_first_party = ["voicevox_engine"]
 isort_known_third_party = ["numpy"]
 [[tool.pysen.lint.mypy_targets]]
-  paths = [".", "voicevox_engine/"]
+paths = ["run.py", "voicevox_engine/"]
```

`paths = ["."]`が正しそうでした！（こうすると全部のPythonファイルの検査をしてくれる）
そうしてみるとテスト周りのコードとかビルド用のコードがエラーになっていました 😇 

ちょっと範囲が広すぎてしまうので、一旦この設定でマージを目指せればいいのかなと思いました！
コメントだけ追加お願いできると！
```suggestion
paths = ["run.py", "voicevox_engine/"] # TODO: paths = ["."]
```